MAUI/ImageEditor/imagefilter.md

```diff
@@ -24,7 +24,7 @@ Using the image editor control, apply various effects such as Brightness, Hue, S
 The EffectValue are the corresponding ImageEffect values, which varies for each effect, and they are explained as follows.

```

Comment:
The `EffectValue` are the corresponding `ImageEffect`(link) values, which vary for each effect and are explained as follows.

Reply:
added